components/family-studies/api/src/main/java/org/phenotips/studies/family/internal/PhenotipsFamilyPermissions.java

```diff
@@ -121,7 +121,13 @@ public void setFamilyPermissionsToCurrentUser(XWikiDocument familyDoc)
         permissions.set(ALLOW, 1, context);
     }
 
-    private String[] getEntitiesWithEditAccessAsString(XWikiDocument patientDoc)
+    /**
+     * Gets a patient document owner permissions.
```

Comment:
I would elaborate a bit more once this is public, since String[] does not give much information about what exactly is returned